english/adipose/awake/awake.md

```diff
@@ -8,4 +8,5 @@ sensation you felt was the star burning your cosmic
 retna. Unfortunitely there is no way to tell if it 
 will be a pleasent dream or a hellish nightmare, but 
 anything is better then your eternal loneliness, so 
-you move on to the white dwarf two parsecs over.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.